man/org.freedesktop.login1.xml

```diff
@@ -156,6 +156,8 @@ node /org/freedesktop/login1 {
       SeatRemoved(s seat_id,
                   o object_path);
       PrepareForShutdown(b start);
+      PrepareForShutdownWithType(b start,
+                                 s type);
```

Comment:
I wonder if it's worth to make this `a{sv}` so that we can extend with more key=value pairs later on without adding yet another signal

Reply:
yeah I think it's worth it as it's simple enough to do